lighthouse-core/report/v2/report-styles.css

```diff
@@ -0,0 +1,160 @@
+/**
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+* {
+  box-sizing: border-box;
+}
+
+.hidden {
+  display: none !important;
+}
+
+/* Text */
+
+.lighthouse-text {
```

Comment:
Agree with having a convention. That said, I don't love any of the options. But I think we need to pick one regardless

Honestly BEM probably has the most tooling support of any CSS convention, so I favor that.

Should we migrate to it in this PR? I could go either way.